src/analysis/retail/priest/holy/modules/core/HolyWordsReductionBySpell.tsx

```diff
@@ -26,9 +26,9 @@ class HolyWordsReductionBySpell extends Analyzer {
 
   constructor(options: Options) {
     super(options);
-    this.lightOfTheNaaruActive = this.selectedCombatant.hasTalent(
-      SPELLS.LIGHT_OF_THE_NAARU_TALENT.id,
-    );
+    //this.lightOfTheNaaruActive = this.selectedCombatant.hasTalent(
+    //  SPELLS.LIGHT_OF_THE_NAARU_TALENT.id,
+    //);
```

Comment:
if this has been removed, you can delete it rather than commenting.

Reply:
I just removed the comment for now but a lot of the file has to be changed for Dragonflight which I have done in a pull request coming later